atlasdb-config/src/main/java/com/palantir/atlasdb/config/TimeLockClientConfig.java

```diff
@@ -27,4 +29,13 @@
     public abstract String client();
 
     public abstract ServerListConfig serversList();
+
+    public ServerListConfig toNamespacedServerList() {
+        return ImmutableServerListConfig.copyOf(serversList())
+                .withServers(serversList()
+                        .servers()
```

Comment:
This looks at first glance like dodgy formatting. Might be more readable to extract the Set of servers as a variable?